### DISCUSSION TITLE
When can Xbox run winui in desktop

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [microsoft/WindowsAppSDK#1568] [Feedback wanted]
## When I used the second preview version of winui 3.0, I found that its performance has been excellent, and it is more and more similar to uwp. I'm wondering when Xbox can run, so that the previously developed uwp can be migrated to winui. What do you think?

**Comment by nickrandolph** [comment]
Latest updates from Microsoft have indicated that Xbox isn't on their immediate roadmap - I'm guessing this means that it's not on roadmap for the next 12 months or so. 
Given there are viable alternatives for UWP for building for Xbox, I can't see adapting WinAppSdk/WinUI to work on xbox being something that will take a priority any time soon.

**Reply by GreenShadeZhang** [reply]
I think uwp and winui desktop will coexist for a long time, but it's really troublesome for people who are new to Windows application development
